app/controllers/consent_forms_controller.rb

```diff
@@ -28,6 +28,10 @@ def record
 
     session.delete(:consent_form_id)
 
-    ConsentFormMailer.confirmation(@consent_form).deliver_later
+    if @consent_form.any_health_answers_truthy?
```

Comment:
Similar comment as the one on the view about tieing `any_health_answers_truthy?` to needing triage in the controller here.